container-host-files/etc/scf/config/scripts/set-opi-registry-port.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+# Locate the kubectl binary.
+kubectl="/var/vcap/packages/kubectl/bin/kubectl"
```

Comment:
Huh, wouldn't munging `$PATH` be easier?

Reply:
I just copied the code from https://github.com/SUSE/scf/blob/develop/container-host-files/etc/scf/config/scripts/configure-az.sh and https://github.com/SUSE/scf/blob/develop/container-host-files/etc/scf/config/scripts/configure-pz.sh to make it look familiar. We could also just hardcode the full path when we call `kubectl` because we only use this once.

What do you prefer?

Reply:
🤷‍♂